content/implementations.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Implementations
+---
+
+TUF provides a framework for integration of the [security](/security)
+properties into new and existing content delivery systems.
+
+While some [adoptions](/adoptions) integrate TUF by implementing the framework
+from scratch, others start from either a TUF [implementation](#implementations)
+or from a TUF [system](#systems).
+
+This page lists open source implementations of TUF which can be used as
+building blocks for any TUF adoption.
+
+## Implementations
+
+TUF implementations provide libraries implementing the primitives and algorithms, such as the detailed client workflow, in the specification.
+
+* [python-tuf](https://github.com/theupdateframework/python-tuf), the reference
+  implementation
+* [go-tuf](https://github.com/theupdateframework/go-tuf/)
+* [tuf-js](https://github.com/theupdateframework/tuf-js)
+
+## Systems
+
+TUF systems build on an implementation and provide an opinionated
+implementation of TUF designed to solve a for a particular use-case.
+
+* [Repository Service for TUF (RSTUF)
+](https://repository-service-tuf.readthedocs.io/en/stable/) is a TUF
+repository designed to integrate into an existing community package repository
+with an established artifact storage and delivery system.
```

Comment:
I'm not sure about "community".
It can be used for private repositories as well.
```suggestion
repository designed to integrate into an existing artifact repository with an
established storage and delivery system.
```

Reply:
LGTM, thanks!